resources/nl-core/ext-Category.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="ext-Category" />
+  <url value="http://nictiz.nl/fhir/StructureDefinition/ext-Category" />
+  <name value="ExtCategory" />
+  <title value="ext Category" />
+  <status value="draft" />
+  <publisher value="Nictiz" />
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="url" />
+      <value value="https://www.nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="An extension that mimics the default FHIR `Resource.category` element that can be used if multiple categories need to be captured, when the cardinality of the core element is 0..1. Resource types where this is the case are for example MedicationAdministration, MedicationDispense, MedicationStatement and Procedure. Note that in the current FHIR R5 build of these resource types, the cardinality of `.category` has been changed to 0..*." />
+  <purpose value="In addition to using this extension, a binding should be added to `extension.valueCodeableConcept` referring to the ValueSet of its respective resource type category and with a binding strength that matches the FHIR core binding strength, along with an appropriate definition and short." />
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <kind value="complex-type" />
+  <abstract value="false" />
+  <context>
```

Comment:
Is this sufficient to use this extension in all the different resources? The `context` seems to be a bit finicky and open to interpretation by validators.

Reply:
Validation does not result in errors, so this context seems to work just fine.